systest/validation/consensus_test.go

```diff
@@ -0,0 +1,198 @@
+package validation
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConsensusValidation(t *testing.T) {
+	type item struct {
+		consensus, state string
+	}
```

Comment:
IMO the unit tests should run with `make test` but they are currently excluded in https://github.com/spacemeshos/go-spacemesh/blob/2ae0df137fe2fcec2ec172a2d040b0f2ac81e2fd/Makefile#L7

Reply:
changed to systest/tests